client/app/utils/places.js

```diff
@@ -0,0 +1,151 @@
+/*
+ Generic utilities for dealing with Google Maps place objects.
+*/
+
+const toRadians = (degrees) => degrees * (Math.PI / 180); // eslint-disable-line no-magic-numbers
+
+/**
+ * Calculate the distance between two points.
+ *
+ * @param {google.maps.LatLng} pointA - first point
+ * @param {google.maps.LatLng} pointB - second point
+ *
+ * @return {Number} distance between the given points (in km?)
+ */
+const computeScale = (pointA, pointB) => {
```

Comment:
Howabout getBoundingboxRadius? We don't need distance between 2 points anywhere atm. (ie. splitting can be done later too if necessary)